.devcontainer/devcontainer.json

```
@@ -24,7 +24,8 @@
 				"ms-azuretools.vscode-docker",
 				"github.vscode-github-actions",
 				"analytic-signal.preview-tiff",
-				"ms-toolsai.jupyter"
+				"ms-toolsai.jupyter",
+				"mechatroner.rainbow-csv"
```

This is a nice extension for csv files; consider adding it.
```suggestion
				"mechatroner.rainbow-csv",
				"janisdd.vscode-edit-csv"
```

---

Sure!

---

Sure!